docs/pages/user-manual.mdx

```diff
@@ -243,40 +233,40 @@ Now `jenkins.pem` can be copied to the jenkins server and passed to `-i`
 equivalent of `tsh login --out` and allows for unrestricted certificate TTL
 values.
 
-## Exploring the Cluster
+## Exploring the cluster
 
 In a Teleport cluster, all nodes periodically ping the cluster's auth server and
 update their status. This allows Teleport users to see which nodes are online
 with the `tsh ls` command:
 
 ```bash
 # This command lists all nodes in the cluster which you previously logged in via "tsh login":
-$ tsh ls
+tsh ls
 
 # Output:
-Node Name     Node ID                Address            Labels
----------     -------                -------            ------
-turing        11111111-dddd-4132     10.1.0.5:3022      os:linux
-turing        22222222-cccc-8274     10.1.0.6:3022      os:linux
-graviton      33333333-aaaa-1284     10.1.0.7:3022      os:osx
+# Node Name     Node ID                Address            Labels
```

Comment:
We no longer show `Node ID`

Reply:
Corrected this and the example below - thanks! 

I will also be factoring most of these sections out and revalidating step by step.